### PR TITLE
Render metric timeseries data

### PR DESCRIPTION
**Pull request by KyleMacPherson**
Format and render the time series data that we request from the cpm api
in order to populate the chart and table components.

This is the first iteration so the designs are not final but provides all the vital
data needed for these components.

### CODE REVIEW

**Comment by chao-xian** [comment]
[(3) Display a chart (unique page views) for a single content item](https://trello.com/c/zms3rqZ1/590-3-display-a-chart-unique-page-views-for-a-single-content-item)

File: app/services/metrics_service.rb [hunk 1]
```diff
@@ -7,12 +7,52 @@ def fetch(base_path:, from:, to:, metric:)
...
+  def keys(data, metric)
+    dates = data[metric].map { |hash| hash['date'] }
+    dates.map { |date| date.last(5) }
+  end
+
+  def values(data, metric)
+    data[metric].map { |hash| hash['value'] }
+  end
+
+  def human_friendly_metric(metric)
```
Comment (chao-xian) [inline app/services/metrics_service.rb]: These private methods can't be tested directly. Another benefit of abstracting these as part of the refactor to presenter classes is that we'll be able to unit test them.

File: app/presenters/chart_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+class ChartPresenter
+  attr_reader :response
```
[the remark this answers is not in the document]
Comment (pmanrubia) [reply app/presenters/chart_presenter.rb]: Could you please add a unit test for this class?

File: spec/features/single_content_item_spec.rb [hunk 3]
```diff
@@ -4,14 +4,36 @@
...
   end
 
-  it 'renders the metric for unique_pageviews' do
+  before do
     content_api_has_metric('base/path',
                            'unique_pageviews',
                            '2000-01-01',
                            '2050-01-01',
                            unique_pageviews: { total: 145_000 })
+    content_api_has_timeseries('base/path',
```
Comment (pmanrubia) [inline spec/features/single_content_item_spec.rb]: Sorry I didn't pick this up before, but can you rename `content_api_has_timeseries` and `content_api_has_metric` to `content_data_api_has*`?

**Pull request merged**